engine/runtime/src/main/java/org/enso/interpreter/runtime/data/Type.java

```diff
@@ -149,15 +152,24 @@ public boolean isBuiltin() {
   }
 
   /**
-   * Returns true iff this type is project-private. A type is project-private iff all its
-   * constructors are project-private. Note that during the compilation, it is ensured by the {@link
+   * Right now types cannot be project private. Waiting for implementation of #8835.
+   *
+   * @return {@code false}
+   */
+  public final boolean isProjectPrivate() {
```

Comment:
Probably remove this method altogether. _project private_ type does not make sense, no?

Reply:
As stated as https://github.com/enso-org/enso/pull/12052#discussion_r1915974359 - once we tackle the implementation of `private` keyword for `type`, it'd be enough to "just" implement this one method.